MainForm.cs

```diff
@@ -0,0 +1,180 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Drawing;
+using System.Windows.Forms;
+
+namespace Task1_GUI
+{
+    public partial class MainForm : Form
+    {
+        public MainForm()
+        {
+            InitializeComponent();
+        }
+
+        public void MainFunc(int combLen1, int combLen2)
+        {
+            Label[] labelsOnCards1 = { c11, c12, c13, c14, c15, c16, c17 };
+            Label[] labelsOnCards2 = { c21, c22, c23, c24, c25, c26, c27 };
+            Panel[] cardPanels1 = { card11, card12, card13, card14, card15, card16, card17 };
+            Panel[] cardPanels2 = { card21, card22, card23, card24, card25, card26, card27 };
+            List<Card> cards1 = new List<Card>();
+            List<Card> cards2 = new List<Card>();
+            int higherCard1_Nominal, higherCardNumber1, higherCard2_Nominal, higherCardNumber2;
+            bool isCombination_1_Correct = true, isCombination_2_Correct = true;
+            bool existIdenticalCards;
+
+            for (int i = 0; i < combLen1; i++)
```

Comment:
Blocks of code from 27-41 lines and 42-56 lines almost equal. We can move it to some method.